trade-finance-manager-ui/templates/utilisation-reports/partials/premium-payments-html.njk

```diff
@@ -39,8 +39,16 @@
         <p class="govuk-body" data-cy="how-to-generate-keying-data-text">
           When payments show as matched, the adjustment data for keying into ACBS will be automatically generated when the 'Generate keying data' button is selected.
         </p>
+
+         <p class="govuk-body" data-cy="how-to-create-record-correction-request-text">
+          If there is an error with the fee record, select the record to create a record correction request to send a query to the bank.
+        </p>
```

Comment:
This needs to be hidden behind the record corrections feature flag so that it is only shown when the button is actually available.

Reply:
done